mappings/net/minecraft/client/font/MultilineText.mapping

```diff
@@ -0,0 +1,46 @@
+CLASS net/minecraft/class_5489 net/minecraft/client/font/MultilineText
```

Comment:
Is this a renderer? or is it a util class for rendering multiple lines of text?

Reply:
It's a class used to render a given text on multiple lines